modules/AWS/eks/main.tf

```diff
@@ -39,4 +39,6 @@ module "eks" {
       capacity_type  = "ON_DEMAND"
     }
   }
+
+  map_roles = var.additional_roles
```

Comment:
I looked into adding unit test for this but seems that the auth-aws value is not accessible in `terratest`.